arelle/plugin/loadFromOIM.py

```diff
@@ -118,6 +118,8 @@
     }
 NSReservedURIAlias = {}
 
+# https://www.xbrl.org/Specification/xbrl-xml/REC-2021-10-13/xbrl-xml-REC-2021-10-13.html#sec-dimensions
+OIMDefaultContextElement = "scenario"
```

Comment:
To be sure, the algorithm doesn't strictly follow the algorithm from the spec in cast the taxonomy intends for segment to be used (and the report would be valid if segment were used), but it also contains unused scenario elements.  That's presumably burdensome to implement, so perhaps just add a small comment for now?

Reply:
Correct. I wrote a comment. Let me know if it makes sense.

We briefly discussed this as a team and decided the complexity wasn't worth the effort right now given that the case is uncommon.